consul/src/main/scala/io/buoyant/consul/v1/ConsulApi.scala

```diff
@@ -161,5 +175,6 @@ case class ServiceNode(
   ServiceName: Option[String],
   ServiceTags: Option[Seq[String]],
   ServiceAddress: Option[String],
-  ServicePort: Option[Int]
+  ServicePort: Option[Int],
+  Status: Option[HealthStatus.Value]
```

Comment:
Ah, I see you've already mentioned that in your PR, sorry – haven't had my coffee yet.